apps/ci-admin/openssh/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG FROM_TAG=""
+FROM alpine:${FROM_TAG}
+
+# Check on 
+# https://pkgs.alpinelinux.org/packages?name=openssh&branch=v3.15
+ARG OPENSSH_VERSION="8.8_p1-r1"
+ENV OPENSSH_VERSION=${OPENSSH_VERSION}
+
+RUN apk --no-cache add openssh=${OPENSSH_VERSION}
+
+COPY uid_entrypoint /usr/local/bin
```

Comment:
you should be able to reuse https://github.com/eclipse-cbi/dockerfiles/blob/master/scripts/uid_entrypoint

Reply:
Fixed